src/styles/styles.js

```diff
@@ -2753,6 +2753,18 @@ const styles = {
         alignSelf: 'center',
     },
 
+    fullScreenTransparentOverlay: {
+        position: 'absolute',
+        width: '100%',
+        height: '100%',
+        top: 0,
+        left: 0,
+        right: 0,
+        bottom: 0,
+        backgroundColor: themeColors.transparentAppBackground,
+        zIndex: 2,
+    },
```

Comment:
I think we can replace these with `absoluteFill` as you are doing on `dropZoneTopInvisibleOverlay`.

```suggestion
    fullScreenTransparentOverlay: {
        ...StyleSheet.absoluteFill
        width: '100%',
        height: '100%',
        backgroundColor: themeColors.dropUIBG,
        zIndex: 2,
    },
```

Reply:
Nope, that breaks display of drop zone